e2e/test/scenarios/dashboard/visualizer/reproductions.cy.spec.ts

```diff
@@ -165,3 +169,124 @@ describe("issue 61521", () => {
     });
   });
 });
+
+describe("issue 65908 (UXW-2293)", () => {
+  const dateParameters: Parameter = {
+    default: "2030-01-01~",
+    id: "d3b78b27",
+    name: "Date Filter",
+    slug: "date_filter",
+    type: "date/all-options",
+  };
+
+  interface CreateDashcardProps {
+    index: number;
+    questionId: number;
+    hideEmptyResults: boolean;
+    withParameterMappings: boolean;
+  }
+  function createDashcard({
+    index,
+    questionId,
+    hideEmptyResults,
+    withParameterMappings,
+  }: CreateDashcardProps): Partial<DashboardCard> {
+    const cardHeightInRows = 10;
+
+    return {
+      col: 0,
+      row: cardHeightInRows * index,
+      size_x: 24,
+      size_y: cardHeightInRows,
+      card_id: questionId,
+      visualization_settings: {
+        "card.hide_empty": hideEmptyResults,
+      },
+      parameter_mappings: withParameterMappings
+        ? [
+            {
+              parameter_id: dateParameters.id,
+              card_id: questionId,
+              target: ["dimension", ["field", ORDERS.CREATED_AT, null]],
+            },
+          ]
+        : undefined,
+    };
+  }
+
+  beforeEach(() => {
+    H.restore();
+    cy.signInAsAdmin();
+
+    const dashboardDetails = {
+      name: "Dashboard with empty result cards",
+      parameters: [dateParameters],
+    } satisfies DashboardDetails;
+
+    const questionDetails: StructuredQuestionDetails = {
+      name: "Orders question",
+      query: {
+        "source-table": ORDERS_ID,
+        limit: 5,
+      },
+    };
+
+    H.createDashboardWithQuestions({
+      dashboardDetails,
+      questions: [questionDetails],
+    }).then(({ dashboard, questions }) => {
+      const [question] = questions;
+      H.updateDashboardCards({
```

Comment:
Adds 4 dashcards, all but the last card connect to the date filter. This way when filtering on the dashboard with a future date that doesn't exist yet, only the last card would be visible.